src/Xrefcheck/Verify.hs

```diff
@@ -154,6 +152,13 @@ data ResponseResult
 
 instance Given ColorMode => Buildable VerifyError where
   build = \case
+    LocalFileDoesNotExist file | '\\' `elem` unRelPosixLink file ->
+      [int||
+      File does not exist:
+        #{file}
+        Its reference contains a backslash. Maybe it uses the wrong path separator.
+      |]
+
```

Comment:
Cannot miss adding a couple of nitpicks )

* As to me, this causes a significant amount of code duplication, I would try to add it differently.
* Also, let's have a dedicated function for `'\\' 'elem' unRelPosixLink file` in `RelPosixLink` utilities, I think this would make certain sense interface-wise.

Reply:
> Mmm, this seems like a justified case to start using OS-dependent bats tests 🤔 Like, set up some env variable in CI config, and rely on it in the test?

I would be worried about some points anyways:
- We should set the corresponding OS env variable for running bats tests in our device.
- Perhaps some Windows user or developer could have some trouble when cloning the repository if it contains such test files. I currently don't have a Windows device for trying it.